util/common_copy_script.py

```diff
@@ -0,0 +1,180 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 20 ('License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#           https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# How to run the script to copy the files:
```

Comment:
Why not use `gsutil cp`?

Is this needed for programatic access?

Reply:
Even for programmatic access, the official GCS Python client already provides an API to copy objects from one bucket to another: https://cloud.google.com/storage/docs/copying-renaming-moving-objects#copy